communication/mavlink_waypoint_handler.cpp

```diff
@@ -186,20 +164,20 @@ void Mavlink_waypoint_handler::receive_count(Mavlink_waypoint_handler* waypoint_
         if (waypoint_handler->waypoint_receiving_ == false)
```

Comment:
It could be due to repeat messages being sent. I do recall one comment in the autopilot that said that QGC sends every message twice. I have not tested it though.